graph-db/serverless.yml

```diff
@@ -69,7 +79,7 @@ resources:
               Resource: !GetAtt CollectionIndexingQueue.Arn
               Condition:
                 ArnEquals:
-                  aws:SourceArn: ${env:CMR_CONCEPT_SNS_TOPIC}
+                  aws:SourceArn: ${self:custom.variables.CMR_CONCEPT_SNS_TOPIC}
```

Comment:
No default value for this is given from `env`. This will cause the deployment to CMR environments to break.